tools/biobb_pytorch/biobb_apply_mdae.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="biobb_pytorch_apply_mdae" name="ApplyMdae" version="4.2.1" >
+    <description>Apply a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+
+    <requirements>
+      <requirement type="package" version="4.2.1">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      ln -s $input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext;
+      ln -s $input_model_pth_path ./input_model_pth_path.$input_model_pth_path.ext;
+      #if str($config_json) != 'None':
+        ln -s $config_json ./config_json.$config_json.ext;
+      #end if
+
+      apply_mdae
+
+      #if str($config_json) != 'None':
+        --config ./config_json.$config_json.ext
+      #end if
+
+      --input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext
+      --input_model_pth_path ./input_model_pth_path.$input_model_pth_path.ext
+      --output_reconstructed_data_npy_path $outname_output_reconstructed_data_npy_path
+      #if str($outname_output_latent_space_npy_path) != 'None':
+        --output_latent_space_npy_path $outname_output_latent_space_npy_path
+      #end if
+      ;
+
+      if test -f $outname_output_reconstructed_data_npy_path; then mv $outname_output_reconstructed_data_npy_path $output_reconstructed_data_npy_path; fi;
```

Comment:
are these lines necessary, can it be that the script doesn't produce an output?

Reply:
No, the script must produce an output but some of the output files are optional. I'll keep it only for the optional outputs

Reply:
You can use the filter tag so that optional outputs are not created, that saves a lot of unneeded empty datasets